proj/cmake/modules/cinderMakeApp.cmake

```diff
@@ -61,13 +65,41 @@ function( ci_make_app )
 		set_source_files_properties( ${ARG_RESOURCES} PROPERTIES HEADER_FILE_ONLY ON MACOSX_PACKAGE_LOCATION Resources )
 	elseif( CINDER_LINUX )
 		unset( ARG_RESOURCES ) # Don't allow resources to be added to the executable on linux
+	elseif( CINDER_MSW )		
+		if( MSVC )
+			# Override the default /MD with /MT
+			foreach( 
+				flag_var
+				CMAKE_C_FLAGS CMAKE_C_FLAGS_DEBUG CMAKE_C_FLAGS_RELEASE CMAKE_C_FLAGS_MINSIZEREL CMAKE_C_FLAGS_RELWITHDEBINFO 
+				CMAKE_CXX_FLAGS CMAKE_CXX_FLAGS_DEBUG CMAKE_CXX_FLAGS_RELEASE CMAKE_CXX_FLAGS_MINSIZEREL CMAKE_CXX_FLAGS_RELWITHDEBINFO 
+			)
+				if( ${flag_var} MATCHES "/MD" )
+					string( REGEX REPLACE "/MD" "/MT" ${flag_var} "${${flag_var}}" )
+					set( "${flag_var}" "${${flag_var}}" PARENT_SCOPE )
+				endif()
+			endforeach()
+			# Force synchronous PDB writes
+			add_compile_options( /FS ) 
+			# Force multiprocess compilation
+			add_compile_options( /MP )
+			# Add lib dirs
+			cmake_policy( PUSH )
+			cmake_policy( SET CMP0015 OLD )
+			link_directories( "${CINDER_PATH}/lib/${CINDER_TARGET_SUBFOLDER}" )
+			cmake_policy( POP )
+		endif()
 	endif()
 
 	add_executable( ${ARG_APP_NAME} MACOSX_BUNDLE WIN32 ${ARG_SOURCES} ${ICON_PATH} ${ARG_RESOURCES} )
 
 	target_include_directories( ${ARG_APP_NAME} PUBLIC ${ARG_INCLUDES} )
 	target_link_libraries( ${ARG_APP_NAME} cinder ${ARG_LIBRARIES} )
 
+	# Ignore Specific Default Libraries
+	if( MSVC )
+		set_target_properties( ${ARG_APP_NAME} PROPERTIES LINK_FLAGS "/NODEFAULTLIB:LIBCMT /NODEFAULTLIB:LIBCPMT" )
```

Comment:
If we set this on the cinder target, won't they get picked up as the app depends on that package? That way we can keep them grouped together.